app/controllers/fhir/r4/api_controller.rb

```diff
@@ -620,6 +726,87 @@ def options
 
   private
 
+  class SaveError < StandardError; end
```

Comment:
This seems kind of lonely here; maybe define this in the ApplicationAPIController instead? I noticed you did that for `ClientError` in https://github.com/SaraAlert/SaraAlert/pull/888.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/870/commits/d27d21a2163d3c613b633349fd3eb75f5fdaccf8